### PR TITLE
[UCSC-82] Link List block pattern

### PR DESCRIPTION
**Pull request by bjcooper**
Adds a block pattern and styles for Link Lists.

Related Ticket: https://moderntribe.atlassian.net/browse/UCSC-82

### CODE REVIEW

**Comment by bjcooper** [comment]
@Vinsanity, looking forward to your feedback here as this is my first block pattern on this project _and_ I'm working from a Miro screenshot instead of a Figma file I can inspect. 😕

File: patterns/link-list.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Title: Link list
+ * Slug: ucsc-2022/link-list
+ * Block Types: core/list
+ * Categories: text, columns
+ */
+?>
+<!-- wp:group {"backgroundColor":"white","textColor":"black","className":"ucsc__link-list","layout":{"type":"constrained"}} -->
+<div class="wp-block-group ucsc__link-list has-black-color has-white-background-color has-text-color has-background"><!-- wp:heading {"lock":{"move":true,"remove":true},"style":{"typography":{"fontStyle":"normal","fontWeight":"700"}},"textColor":"black","className":"ucsc__link-list-header","fontSize":"two"} -->
```
Comment (Vinsanity) [inline patterns/link-list.php:10]: Let's move this to the group wrapper `ucsc__link-list` so all elements can be controlled if need be.

Comment (bjcooper) [reply patterns/link-list.php:10]: That class is on the outer-most HTML element of the block, @Vinsanity . I'm not sure what you mean.

**Review by Vinsanity — APPROVED** [review]
🟢  Nice work here again, Brett. Ship it!

**Pull request merged**